tools/binlog/deploy.md

```diff
@@ -0,0 +1,508 @@
+        ```
+
+    **Method #2**: Deploy a TiDB cluster containing Pump from scratch.
+
+    For how to use Ansible to deploy the TiDB cluster, see [Deploy TiDB Using Ansible](../../dev/how-to/deploy/orchestrated/ansible.md).
+
+3. Check the Pump status.
+
+    Use `binlogctl` to check the Pump status. Change the `pd-urls` parameter to the PD address of the cluster. If `State` is `online`, Pump is started successfully.
+
+    ```bash
+    $ cd /home/tidb/tidb-ansible
+    $ resources/bin/binlogctl -pd-urls=http://172.16.10.72:2379 -cmd pumps
+    
+    INFO[0000] pump: {NodeID: ip-172-16-10-72:8250, Addr: 172.16.10.72:8250, State: online, MaxCommitTS: 403051525690884099, UpdateTime: 2018-12-25 14:23:37 +0800 CST}
+    INFO[0000] pump: {NodeID: ip-172-16-10-73:8250, Addr: 172.16.10.73:8250, State: online, MaxCommitTS: 403051525703991299, UpdateTime: 2018-12-25 14:23:36 +0800 CST}
+    INFO[0000] pump: {NodeID: ip-172-16-10-74:8250, Addr: 172.16.10.74:8250, State: online, MaxCommitTS: 403051525717360643, UpdateTime: 2018-12-25 14:23:35 +0800 CST}
+    ```
+
+### Step 3: Deploy Drainer
+
+1. Obtain `initial_commit_ts`. 
+
+    Run the following command to use `binlogctl` to generate the `tso` information which is needed for the initial start of Drainer:
+
+    ```bash
+    $ cd /home/tidb/tidb-ansible
+    $ resources/bin/binlogctl -pd-urls=http://127.0.0.1:2379 -cmd generate_meta
+    INFO[0000] [pd] create pd client with endpoints [http://192.168.199.118:32379]
+    INFO[0000] [pd] leader switches to: http://192.168.199.118:32379, previous:
+    INFO[0000] [pd] init cluster id 6569368151110378289
+    2018/06/21 11:24:47 meta.go:117: [info] meta: &{CommitTS:400962745252184065}
+    ```
+
+    This command outputs `meta: &{CommitTS:400962745252184065}`, and the value of `CommitTS` is used as the value of the `initial-commit-ts` parameter needed for the initial start of Drainer.
+
+2. Modify the `tidb-ansible/inventory.ini` file.
+
+    Add the deployment machine IPs for `drainer_servers`. Set `initial_commit_ts` to the value you have obtained, which is only used for the initial start of Drainer.
+
+    - Assume that the downstream is MySQL with the alias `drainer_mysql`:
+
+        ```ini
+        [drainer_servers]
+        drainer_mysql ansible_host=172.16.10.71 initial_commit_ts="402899541671542785"
+        ```
+
+    - Assume that the downstream is `file` with the alias `drainer_file`:
+
+        ```ini
+        [drainer_servers]
+        drainer_file ansible_host=172.16.10.71 initial_commit_ts="402899541671542785"
+        ```
+
+3. Modify the configuration file.
+
+    - Assume that the downstream is MySQL:
+
+        ```bash
+        $ cd /home/tidb/tidb-ansible/conf
+        $ cp drainer.toml drainer_mysql_drainer.toml
+        $ vi drainer_mysql_drainer.toml
+        ```
+
+        > **Note:** Name the configuration file as `alias_drainer.toml`. Otherwise, the customized configuration file cannot be found during the deployment process.
```

Comment:
```suggestion
        > **Note:**
        >
        > Name the configuration file as `alias_drainer.toml`. Otherwise, the customized configuration file cannot be found during the deployment process.
```